### ISSUE TITLE
Question about Serene Seasons day length

### ISSUE THREAD

**Issue by Biokoro** [DuckyCrayfish/hourglass#47]
If I were to set the speed of both day and night to 0.5 (making everything twice as long) do I also need to double the "day_duration" setting in the Serene Seasons config, or does this scale automaticaly with Hourglass?

**Comment by DuckyCrayfish** [comment]
I'm actually curious myself. Did you ever figure out the answer?

**Comment by SaltSouls** [comment]
I've actually looked into this(Kinda). Hourglass doesn't actually change the amount of time in the day, but rather slows down how fast it progresses.
Example:
Hourglass with day and night set to 0.5 is still 24000(same as vanilla).
Serene Seasons set to double is 48000.
Hourglass just extends how long it takes to progress time, while Serene Seasons literally increases the amount of time.

TLDR: No you don't need to change anything with serene seasons.

**Comment by VinterNacht** [comment]
I know this is supposed to be closed, but these answers will likely be looked for together in the future (Just as they are now).  Does the variation is light/dark throughout the seasons with Serene Seasons also occur with Hourglass?
